src/utils/splitStringToXChar.ts

```diff
@@ -0,0 +1,3 @@
+export const sliceStrToXChar = (s: string, len: number): string => {
+  return s.length < len ? s : s.slice(0, len - 3) + '...';
+};
```

Comment:
export const sliceStrToXChar = (s: string, len: number): string =>
  s.length <= len ? s : `${s.slice(0, len - 3)}...`;

-> template string
-> also allow title if it has 45 characters